spec/models/account_moderation_note_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe AccountModerationNote do
+  describe 'latest scope' do
+    it 'returns account moderation notes oldest to newest' do
+      account = Fabricate(:account)
+      note1 = Fabricate(:account_moderation_note, target_account: account)
+      note2 = Fabricate(:account_moderation_note, target_account: account)
+
+      expect(account.targeted_moderation_notes.latest).to eq [note1, note2]
```

Comment:
This might look a little weird, but there's two associations here: `account_moderation_notes` and `targeted_moderation_notes` — the latter is what's actually used on the Account details page in the Moderation UI, the former is just an association of all account moderation notes created by a given account.